docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ax = perf_df.plot.bar(color='#7400ff', ylim=(1, 100000), rot=0, xlabel="UDFs", ylabel="Speedup factor")
</pre><p>groupby_udf is 458x speedup with overhead and 415x speedup without overhead. Why? Is there overhead for the CPU code that is being eliminated too? I expected that the "without overhead" would be strictly faster.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
I initially thought there was JIT overhead & caching with UDF's in groupby too. @brandon-b-miller is working on enabling this currently. So separated out groupby UDF performance comparison into a new section.